components/Pages/CarePackages/CorePackageDetails/FurtherDetails.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { Checkbox } from 'components/HackneyDS';
+import { Controller } from 'react-hook-form';
+import FormGroup from 'components/HackneyDS/FormGroup';
+
+export const FurtherDetails = ({ settings, control, setValue }) => {
+  const checkboxOptions = [
```

Comment:
Let's move this outside of component in order to prevent recreation every render